source/settings/pigeonhole/variables.rst

```diff
@@ -0,0 +1,29 @@
+=====================================
+Pigeonhole Sieve: Variables Extension
+=====================================
+
+.. _plugin-sieve-setting-sieve_variables_max_scope_size:
+
+``sieve_variables_max_scope_size``
+----------------------------------
+
+ - Default: ``255``
+ - Value: :ref:`uint`
+
+.. versionadded:: v0.5.0
+
+The maximum number of variables that can be declared in a scope. There are currently two variable scopes:
+the normal script scope and the global scope created by the :ref:`include <pigeonhole_extension_include>`. The minimum value for this setting is 128. 
```

Comment:
reference title is just "include" which doesn't work in this sentence. should be "include extension"